src/apivip_check/apivip_check.go

```
@@ -38,14 +35,6 @@ func CheckAPIConnectivity(checkAPIRequestStr string, log logrus.FieldLogger) (st
 		return createResponse(false), wrapped.Error(), 0
 	}
 
-	if checkAPIRequest.VerifyCidr {
-		if err := verifyCIDR(*checkAPIRequest.URL, log); err != nil {
```

Replace this with an error log, rather than removing it altogether

---

The verification happens at the service level. Switching this to an error will make it fail for all deployments wen `SUPPORT_L2=True`

---

I didn't mean literal error, just error log. If the service passing this VerifyCidr flag and the flag does nothing, I prefer we have some indication the service is doing "nonsensical" things and that we should fix the service to stop doing those nonsensical things

---

The flag will be removed, there's a PR for it already: https://github.com/openshift/assisted-service/pull/2650

---

I understand, but it's still a parameter [defined](https://github.com/openshift/assisted-service/blob/d8888171f88fd2ab69920e2088f742fd35fa92ee/swagger.yaml#L7592-L7594) in the swagger and it's still part of our service-agent API, and I prefer to have an error/warning log when that parameter is set to true rather than silently ignoring it, in case someone tries to use it in the future for whatever reason

---

Oh, I now see where you are coming from. Considering this validation as an API, and the inputs it receives (as defined in `swagger`).

with that in mind, I think we may want to hold off with removing this functionality until we completely remove the endpoint from the service. What do you think?